src/chainstate/burn/db/sortdb.rs

```diff
@@ -699,6 +731,21 @@ const SORTITION_DB_SCHEMA_4: &'static [&'static str] = &[
 // update this to add new indexes
 const LAST_SORTITION_DB_INDEX: &'static str = "index_burn_header_hash_pox_valid";
```

Comment:
Please change this constant to `index_peg_in_burn_header_hash`.

Reply:
Done ✅